### ISSUE TITLE
There may be one 'bug'  in  models.py ?

### ISSUE THREAD

**Issue by smallreed** [sxyu/pixel-nerf#68]
In the file 'models.py’,  the following code seems wrong:
# Grab encoder's latent code.
uv = -xyz[:, :, :2] / xyz[:, :, 2:]  # (SB, B, 2)

The 'correct' one should be：
# Grab encoder's latent code.
#uv = -xyz[:, :, :2] / xyz[:, :, 2:]  # (SB, B, 2)
h = xyz[:, :, 1:2] / xyz[:, :, 2:]
w = -xyz[:, :, 0:1] / xyz[:, :, 2:]
uv = torch.cat([h,w], dim = -1)

Looking forward to receiving a response from the author as soon as possible.
